styles/base.css

```diff
@@ -2,8 +2,11 @@
 *::before,
 *::after {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
 ul[role="list"] {
   list-style: none;
+  padding: 0;
```

Comment:
I think this padding is unnecessary, since padding 0 is already defined above with the universal selector.

Reply:
Yes, forgot to remove it.